SharedSources/MediaLibraryModel/AlbumModel.swift

```diff
@@ -41,9 +41,10 @@ class AlbumModel: MLBaseModel {
 
 // MARK: - Sort
 extension AlbumModel {
-    func sort(by criteria: VLCMLSortingCriteria) {
-        files = medialibrary.albums(sortingCriteria: criteria)
+    func sort(by criteria: VLCMLSortingCriteria, desc: Bool) {
+        files = medialibrary.albums(sortingCriteria: criteria, desc: desc)
         sortModel.currentSort = criteria
+        sortModel.desc = desc
```

Comment:
I agree that it would be better but it wouldn't change fix the issue.

I think the whole sortModel logic should be reconsidered. Indeed, I do find the fact that we need to set in each model ungraceful.

Hoping that you don't mind, I will merge this, create a ticket and address the refactoring in another time.